src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -302,6 +302,9 @@
   <data name="PowerLauncher_ClearInputOnLaunch.Content" xml:space="preserve">
     <value>Clear the previous query on launch</value>
   </data>
+  <data name="PowerLauncher_Theme.Text" xml:space="preserve">
+    <value>Choose color</value>

```

Comment:
I think here we should use the same text like on the other settings pages.

Reply:
Strings should be localisable

Reply:
@aron-junker
This string could be localized. It is within resource file.

Reply:
Ok my mistake. I'm only a triager.😅